project/Dependencies.scala

```diff
@@ -72,7 +72,7 @@ object Dependencies {
     val kryo = "com.esotericsoftware" % "kryo" % kryoVersion
     // exclude jar "com.esotericsoftware.kryo % kryo" which conflicts with spark 3.0
     val xgboostDep = "ml.dmlc" %% "xgboost4j" % xgboostVersion exclude("com.esotericsoftware.kryo", "kryo")
-    val xgboostPredictorDep = "biz.k11i" % "xgboost-predictor" % "0.3.1" exclude("com.esotericsoftware.kryo", "kryo")
+    val xgboostPredictorDep = "ai.h2o" % "xgboost-predictor" % "0.3.17" exclude("com.esotericsoftware.kryo", "kryo")
```

Comment:
I saw the last update from https://github.com/komiya-atsushi/xgboost-predictor-java was Nov 26, 2019. Does H2O's fork contain changes that won't go back to upstream?

cc: @komiya-atsushi @honzasterba

Reply:
Hi, I did not plan to bring our changes to the original repo. I think the two diverged a little even before I started working on H2O.

Reply:
@komiya-atsushi Do you want to support xgboost 1.0 in your repo ?

Reply:
I merged the change into spark-3.0.0 branch. We might want to switch back to the official release once it supports xgboost 1.0.0.